README.md

```diff
@@ -4,22 +4,96 @@ This project is a ActiveMQ Artemis Self Provisioning Plugin to the Administrator
 
 ## Local development
 
+To be able to run the local development environment you need to:
+* have access to a local or remote OpenShift cluster
+* have the operator installed on the cluster
+* have the plugin running
+* have the console running
+
+### Setting up an OpenShift cluster
+
+In order to run the project you need to have access to an OpenShift cluster.
+If you don't have an access to a remote one you can deploy one on your machine
+with `crc`.
+
+#### Local cluster
+
+Follow the documentation:
+https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.34/html-single/getting_started_guide/index#introducing
+
+> [!WARNING]
+> If you're encountering an issue where `crc` gets stuck in the step `Waiting
+> for kube-apiserver availability` or `Waiting until the user's pull secret is
+> written to the instance disk...` [you might
+> need](https://github.com/crc-org/crc/issues/4110) to
+> configure the network as local: `crc config set network-mode user`
+
+Once your environment is set up you simply need to `crc start` your cluster.
+
+#### Connecting to the cluster
+
+Depending on the remote or local env:
+
+* `oc login -u kubeadmin
+https://api.ci-ln-x671mxk-76ef8.origin-ci-int-aws.dev.rhcloud.com:6443` (to
+adapt depending on your cluster address)
+* `oc login -u kubeadmin https://api.crc.testing:6443`
+
+### Installing the operator
+
+The plugin requires having access to the operator to function. You can either
+get the operator from the hub or from the upstream repo.
```

Comment:
'from the hub' -> 'from operatorhub? Do we need to differentiate between the ocp local operatorhub and operatorhub.io given that we aren't available yet via the community operatorhub.io but are via the internal operatorhub.